### PR TITLE
Captain v1.9.11 - Release Candidate

### PR DESCRIPTION
**Pull request by pierrebeaucamp**
Note: As this is our first release on `releases/v1`, the diff & commit history on this PR are not complete. To see all changes from the previous release, go to https://github.com/rwx-research/captain/compare/e14a5b9...b0f3f7d9af93c66f62a2e0b6f8546b899ec7dd75

Release notes (to be published) are below:

---

### Added

- Ability to explicitly disable Cloud mode through the config file.
- Additional warning messages if Captain is misconfigured.
- A fallback to the pull-request name / number if Captain can't determine a commit message in GitHub Actions.
- Ability to manually set a "title" for a test-suite execution in Captain via `CAPTAIN_TITLE` or `--title`. This can be useful if Captain is unable to derive commit messages from your environment.
- Additional options to configure the partitioning delimiter (can now be set via `CAPTAIN_DELIMITER`)

### Changed

- Some error messages were modified to be easier to understand.

### CODE REVIEW

File: version.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
 package captain
 
 // Version is the current version of this package. This needs to get bumped with every merge to 'main'.
-const Version = "v1.9.10"
+const Version = "v1.9.11"
```
Comment (TAGraves) [inline version.go:4]: Is that idea that we merge releases/v1 into main after this so that the version is updated on main?

**Comment by kylekthompson** [comment]
> Changelog

Is this missing @michaelglass's bug fix for captain add/remove?

**Comment by kylekthompson** [comment]
Specifically- https://github.com/rwx-research/captain/pull/3

File: version.go (hunk 1 above)
Comment (pierrebeaucamp) [reply version.go:4]: We could. But I was thinking that `main` should be changed to a different or invalid version number instead so `captain --version` from main doesn't output a version that aligns with a particular release.

The simplest would probably be to have main on the "next" version with a `-dev` suffix (like `v1.9.12-dev`)

**Comment by pierrebeaucamp** [comment]
> Is this missing @michaelglass's bug fix for captain add/remove?

No, but that's for double-checking. I did not include it because I don't understand what exactly was fixed there. It looks like `add/remove` previously explicitly required a `--suite-id` and now it can also be invoked with the suite-id as an argument instead? We could add this to the "added" section, I suppose.

Before we merge, we should ideally also update the documentation & help messages. `captain add flake --help` and co. still show the old examples.

**Comment by pierrebeaucamp** [comment]
Works for me

**Comment by pierrebeaucamp** [comment]
The changelog needs to be set manually and can be edited on the releases page

File: version.go (hunk 1 above)
Comment (michaelglass) [reply version.go:4]: IIRC abq uses `git describe` to tag in-between versions

**Comment by michaelglass** [comment]
@pierrebeaucamp yeah #3 allows positional suite-id for `git add` and `git remove`

**Comment by pierrebeaucamp** [comment]
Updated the changelog. Thanks

**Pull request merged**